test/unit/validations/content.spec.ts

```diff
@@ -29,7 +29,16 @@ describe('Content', () => {
 
   it(`When a hash that was not uploaded was already stored, then no error is returned`, async () => {
```

Comment:
nit: Can we work in the test name :P? Had a hard time understanding it.
~Something like:
`When a hash content file was not uploaded BUT was already stored, ...`